package.json

```diff
@@ -67,6 +67,7 @@
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "html-webpack-plugin": "^5.5.3",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
apparently this became necessary with v28, I had to do this to get around `TypeError: Cannot read properties of undefined (reading 'html')`

https://stackoverflow.com/a/72180987